graf3d/eve7/src/REveManager.cxx

```diff
@@ -917,14 +918,14 @@ void REveManager::PublishChanges()
    fScenes->ProcessSceneChanges();
    jobj["content"] = "EndChanges";
 
-   if (!fLogger.fLogEntries.empty()) {
+   if (!gEveLogEntries.empty()) {
 
       constexpr static int numLevels = static_cast<int>(ELogLevel::kDebug) + 1;
       constexpr static std::array<const char *, numLevels> sTag{
          {"{unset-error-level please report}", "FATAL", "Error", "Warning", "Info", "Debug"}};
 
       std::stringstream strm;
-      for (auto entry : fLogger.fLogEntries) {
+      for (auto entry : gEveLogEntries) {
```

Comment:
It is better to use `&` in the loop `for (auto &entry : gEveLogEntries)` to avoid unnecessary object copy